packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -223,6 +223,7 @@ abstract class MultiDragGestureRecognizer extends GestureRecognizer {
     )
     super.kind,
     super.supportedDevices,
+    this.inputPointerBehavior,
```

Comment:
inputPointerBehavior so generic, it really could refer to so many things. Maybe something like `isButtonAllowed`, `areButtonsAllowed`, or `buttonAcceptBehavior`?

Reply:
I agree. Source: https://github.com/flutter/flutter/pull/111852#discussion_r986298568.

I'll try one of the names you suggested.

Reply:
The hard challenge is "button" doesn't convey a lot of meaning.

Reply:
This is in the `GestureRecognizer` level, not the framework / `GestureDetector` level. So I think button is pretty clearly relating to a mouse (pointer) button rather than some widget. Maybe specifically using button**s** instead of button will associate it more with the `buttons` field on `PointerEvent`?

Reply:
`isPointerAllowed` sounds good to me. https://stackoverflow.com/questions/8548232/whats-the-best-way-to-name-a-predicate-function